src/test/groovy/com/palantir/gradle/revapi/RevapiSpec.groovy

```diff
@@ -27,6 +28,7 @@ class RevapiSpec extends IntegrationSpec {
 
```

Comment:
I locally ran tests (just gradle clean build) and got one failure. Is it flaky? Can you please check?

<img width="1258" alt="image" src="https://github.com/revapi/gradle-revapi/assets/5889404/dc55a76d-e44f-4504-b110-b7f9496571d9">

Reply:
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at com.google.common.collect.RegularImmutableList.get(RegularImmutableList.java:77)
	at com.palantir.gradle.revapi.RevapiExtension.oldGroupNameVersion(RevapiExtension.java:72)
	at com.palantir.gradle.revapi.RevapiAcceptBreakTask.oldGroupNameVersion(RevapiAcceptBreakTask.java:97)
	at com.palantir.gradle.revapi.RevapiAcceptBreakTask.lambda$addVersionOverride$1(RevapiAcceptBreakTask.java:81)
	at com.palantir.gradle.revapi.ConfigManager.modifyConfigFile(ConfigManager.java:41)
	at com.palantir.gradle.revapi.RevapiAcceptBreakTask.addVersionOverride(RevapiAcceptBreakTask.java:80)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:125)

Reply:
this passed locally for me

Reply:
@jbonofre: Can you please run this PR locally and confirm the results?

Reply:
Let me try

Reply:
@nastra: For me it is always failing. I debugged the failed test `RevapiSpec.accepting breaks individually should work`, it looks like `oldVersions` list is empty in `RevapiExtension`. 

It tries to fetch old version from git tags. I don't have any tags. Do you locally have any tags?
Whats the output of `git fetch --all --tags && git tag -l`?

Reply:
We discussed offline and it is a legit failure because it was looking for tags. 

It was passing locally for nastra because he as having a remote from old fork also added which fetching some tags.
Ignored the test it now.